discovery/file/file.go

```diff
@@ -33,6 +34,54 @@ import (
 
 const fileSDFilepathLabel = model.MetaLabelPrefix + "filepath"
 
+//TimestampCollector is a Custom Collector for Timestamps of the files
```

Comment:
The comment formatting style in Prometheus projects to use full sentences, always including the trailing dot and to leave a space to the comment start marker. Please update accordingly everywhere.